packages/@stylexjs/postcss-plugin/README.md

```diff
@@ -130,11 +130,16 @@ and specify desired options. Refer to
 ### useCSSLayers
 
 ```js
-useCSSLayers: boolean; // Default: false
+useCSSLayers: 'none' | 'native' | 'polyfill'; // Default: 'none'
 ```
 
-Enabling this option switches Stylex from using `:not(#\#)` to using `@layers`
-for handling CSS specificity.
+The `useCSSLayers` option controls how StyleX handles CSS specificity and layer
+management. It supports three strategies:
+
+- `'none'` (default): Uses `:not(#\#)` to handle CSS specificity without layers
```

Comment:
`none` should not add layers at all and rely only on generated ordering using the "priority"